### PR TITLE
Move to Serilog And Wire Client to it

### PR DESCRIPTION
**Pull request by thefringeninja**

### CODE REVIEW

File: src/EventStore.Core/Services/Gossip/GossipServiceBase.cs [hunk 1]
```diff
@@ -332,37 +332,20 @@ public abstract class GossipServiceBase : IHandle<SystemMessage.SystemInit>,
...
-				);
-			}
+			List<MemberInfo> oldMembers = oldCluster.Members
+				.OrderByDescending(x => x.InternalHttpEndPoint, ipEndPointComparer).ToList();
+			List<MemberInfo> newMembers = newCluster.Members
+				.OrderByDescending(x => x.InternalHttpEndPoint, ipEndPointComparer).ToList();
+			Log.Debug(
+				"CLUSTER HAS CHANGED {source}"
+				+ "\nOld:"
+				+ "\n{@oldMembers}"
```
Comment (pgermishuys) [inline src/EventStore.Core/Services/Gossip/GossipServiceBase.cs]:
```suggestion
				+ "\n{@oldMembers}"
```
```suggestion
				+ "\n{oldMembers}"
```

File: src/EventStore.Core/Services/Gossip/GossipServiceBase.cs [hunk 2]
```diff
@@ -332,37 +332,20 @@ public abstract class GossipServiceBase : IHandle<SystemMessage.SystemInit>,
...
+				+ "\nNew:"
+				+ "\n{@newMembers}"
```
Comment (pgermishuys) [inline src/EventStore.Core/Services/Gossip/GossipServiceBase.cs]:
```suggestion
				+ "\n{newMembers}"
```

File: src/EventStore.Core/Services/Gossip/GossipServiceBase.cs (hunk 1 above)
Comment (pgermishuys) [inline src/EventStore.Core/Services/Gossip/GossipServiceBase.cs]:
```suggestion
				+ "\n{oldMembers}"
```

File: src/EventStore.TestClient/Commands/WriteFloodProcessor.cs [hunk 3]
```diff
@@ -121,7 +121,7 @@ internal class WriteFloodProcessor : ICmdProcessor {
 						if (localAll % 100000 == 0) {
 							var elapsed = sw2.Elapsed;
 							sw2.Restart();
-							context.Log.Trace(
+							context.Log.Verbose(
```
Comment (hayley-jean) [inline src/EventStore.TestClient/Commands/WriteFloodProcessor.cs]:
```suggestion
							context.Log.Debug(
```

If we leave this on verbose, it won't get logged out

File: src/EventStore.TestClient/Commands/WriteFloodClientApiProcessor.cs [hunk 4]
```diff
@@ -96,7 +96,7 @@ internal class WriteFloodClientApiProcessor : ICmdProcessor {
 							if (localAll % 100000 == 0) {
 								var elapsed = sw2.Elapsed;
 								sw2.Restart();
-								context.Log.Trace("\nDONE TOTAL {writes} WRITES IN {elapsed} ({rate:0.0}/s).",
+								context.Log.Verbose("\nDONE TOTAL {writes} WRITES IN {elapsed} ({rate:0.0}/s).",
```
Comment (hayley-jean) [inline src/EventStore.TestClient/Commands/WriteFloodClientApiProcessor.cs]:
```suggestion
								context.Log.Debug("\nDONE TOTAL {writes} WRITES IN {elapsed} ({rate:0.0}/s).",
```

File: src/EventStore.TestClient/Commands/SubscriptionStressTestProcessor.cs [hunk 5]
```diff
@@ -43,7 +43,7 @@ internal class SubscriptionStressTestProcessor : ICmdProcessor {
 						var c = Interlocked.Increment(ref appearedCnt);
 						if (c % 1000 == 0) Console.Write('\'');
 						if (c % 100000 == 0) {
-							context.Log.Trace("Received total {events} events ({rate} per sec)...", c,
+							context.Log.Verbose("Received total {events} events ({rate} per sec)...", c,
```
Comment (hayley-jean) [inline src/EventStore.TestClient/Commands/SubscriptionStressTestProcessor.cs]:
```suggestion
							context.Log.Debug("Received total {events} events ({rate} per sec)...", c,
```

**Pull request merged**